runtime/common/src/auctions.rs

```diff
@@ -1805,7 +1806,8 @@ mod benchmarking {
 			// Create a new auction
 			let duration = T::BlockNumber::max_value();
 			let lease_period_index = LeasePeriodOf::<T>::zero();
-			let origin = T::InitiateOrigin::successful_origin();
+			let origin = T::InitiateOrigin::try_successful_origin()
+				.expect("InitiateOrigin has no successful origin required for the benchmark");
```

Comment:
Why do you sometimes expect and sometimes return an error?

Reply:
Yeah, there seem to be a lot of theses cases.

Reply:
Here `expect` because `InitiateOrigin` is not an origin of `bid` extrinsic. I do not wanna add here unexpected behaviour.
[Here](https://github.com/paritytech/polkadot/pull/6598/files#diff-0007716ce0cd796e5f500474ccea38bfa1dd33dbc1b2af545f08eb5dd7a65a75R1030) for example, its an origin of the extrinsic the bench written for, so its `Weightless` if no origin. 

There are only these two cases in related PRs.

Reply:
Ahh!

Reply:
These benchmarks should probably be refactored in a way that an auction can be scheduled by calling some non-dispatchable function.

Reply:
But yeah, not required by this pr